.travis.yml

```diff
@@ -108,6 +108,7 @@ after_failure:
 
 script:
   - autoreconf -fvi
-  - CFLAGS="$CFLAGS -Werror" ./configure
+  - CFLAGS="$CFLAGS -Werror" ./configure --prefix="$HOME/travis_build"
```

Comment:
Maybe you could change it to just `--prefix="$HOME"` since that's what I use in all of the `install` scripts.